dependencies/prepare.sh

```
@@ -27,7 +28,7 @@ apt-get install -y \
   cmake
 
 dpkg-architecture -a "$DEBIAN_ARCH" >/env
-echo "PKG_CONFIG_PATH=/usr/lib/${DEB_HOST_MULTIARCH}/pkgconfig" >>/env
-# shellcheck disable=SC2046
-export $(cat /env)
+export $(cat /env) # Set the proper DEB_HOST_MULTIARCH
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)